pytorch/training/docker/1.12/py3/cu113/Dockerfile.sagemaker.gpu.os_scan_allowlist.json

```diff
@@ -50,11 +80,11 @@
       "cvss_v31_score": 9.8,
       "cvss_v2_score": 0,
       "cvss_v3_severity": "CRITICAL",
-      "source_url": "https://people.canonical.com/~ubuntu-security/cve/2022/CVE-2022-45907.html",
-      "source": "UBUNTU_CVE",
-      "severity": "MEDIUM",
+      "source_url": "https://snyk.io/vuln/SNYK-PYTHON-TORCH-3149871",
+      "source": "SNYK",
+      "severity": "CRITICAL",
       "status": "ACTIVE",
-      "title": "CVE-2022-45907 - torch"
+      "title": "IN1-PYTHON-TORCH-3149871 - torch"
     }
   ]
 }
```

Comment:
Mind adding a new line under here for sanity? Notice the red circle on the end of the file?